bin/activate-hermit

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+# This file must be used with "source bin/activate-hermit" from bash or zsh.
```

Comment:
@phoebe-lew I guess this doesn't support `fish` shell but sounds like its possible to use:

https://cashapp.github.io/hermit/faq/#fish-support-via-direnv